packages/ember-glimmer/tests/integration/helpers/input-test.js

```diff
@@ -0,0 +1,444 @@
+import { set } from 'ember-metal/property_set';
+import TextField from 'ember-templates/components/text_field';
```

Comment:
please use the helpers, these become const with the flag and we run both @glimmer and @htmlbars without the flag.